### PR TITLE
Fix url spec

### PR DESCRIPTION
**Pull request by rndmh3ro**
replace customized url_spec with native one.

### CODE REVIEW

File: plugins/modules/icinga_endpoint.py [hunk 1]
```diff
@@ -34,7 +34,7 @@
 description:
    - "Add or remove an endpoint to Icinga2 through the director API."
 author: Aaron Bulmahn (@arbu)
-extends_documentation_fragment: t_systems_mms.icinga_director.auth_options
+extends_documentation_fragment: url
```
Comment (schurzi) [inline plugins/modules/icinga_endpoint.py:37]: should this rather be a fully qualified name? e.g. `ansibl.base.url` or something like that?

Comment (rndmh3ro) [reply plugins/modules/icinga_endpoint.py:37]: Yes. I should have not looked at other collections. :)

File: plugins/modules/icinga_command.py [hunk 2]
```diff
@@ -183,13 +184,9 @@
 def main():
     # use the predefined argument spec for url
     argument_spec = url_argument_spec()
-    # remove unnecessary argument 'force'
-    del argument_spec["force"]
-    del argument_spec["http_agent"]
     # add our own arguments
     argument_spec.update(
         state=dict(default="present", choices=["absent", "present"]),
-        url=dict(required=True),
```
Comment (felixfontein) [inline plugins/modules/icinga_command.py:192]: Please note that `url` is not required by `url_argument_spec()`, so keeping this here is a good idea.

Comment (rndmh3ro) [reply plugins/modules/icinga_command.py:192]: I'll fix that.

**Pull request merged**